ui/plugin-system/src/runtime/template-variables.ts

```diff
@@ -60,3 +60,11 @@ export function useTemplateVariableValues(names?: string[]) {
 
   return values;
 }
+
+// Convenience hook for replacing template variables in a string
+export function useReplaceVariablesInString(str: string | undefined): string | undefined {
```

Comment:
Move this from `@dashboards` package to `@plugin-system` since its a convenient helper to replace variables with one line.